### DISCUSSION TITLE
Home Router resets IP address of Edge.  UWA losses connection

### DISCUSSION THREAD

**Discussion by jjsenay** [ambianic/ambianic-ui#790] [Q&A] [answered]
I could connect to the Edge this morning using the beta UI-PWA .  Went out and did some shopping, came back and I could not get to the timeline.  

I tried to reconnect via the beta UI-PWA but there was not a **clear** way to do it.  

I then tried to ssh into the Edge and I could not.  I tried to ping the Edge, but I could not.  Opened up my router admin page and saw the Edge was giving a new IP address.

My home router, every month ( I think ) reassigns a new IP address.  The beta PWA-UI losses connection to the EDGE.
I have to reload the beta PWA-UI and since I am on a VPN I use connect to remote option and I connect to the Edge Timeline.

How to fix?  On the home router assign an IP address to each EDGE.  Do not let the DHCP server regroom/reset the IP address of the Edge.

**Comment by ivelin** [comment]
New Ip should not be an issue. Can you do a screen recording of what
exactly happens on the client side with dev console open.

**Comment by vickywane** [comment]
@jjsenay I’m waiting for your recording to take a look at this.

**Comment by ivelin** [comment]
Is this still an issue?

**Reply by jjsenay** [reply]
All,
It is not the fault of the Edge but a function of my Internet Router.  Most routers give out IP addresses on the LAN via DHCP.  Every once in a while some Internet routers will reset the IP addresses on the LAN.  Thats what my router did.  All routers should have the ability to reserve an IP address by MAC address.

I went to my routers LAN setup and found the section where IP addresses are reserved by MAC address.  I reserved the IP addresses that the Edges where using.  No problems since.
![LAN_Router_Setup](https://user-images.githubusercontent.com/8701163/145627389-0971e383-8564-4b77-b566-749f5cadb92f.png)

**Reply by ivelin** [reply]
Cool. Thanks for sharing.

**Comment by ivelin** [comment, marked as answer]
Answer above. ^
